datamade/bga-payroll
# Update pre-processing data pipeline to accommodate pay breakout

## Description

This pull request:

- Removes the pre-processing step to sum base and extra pay
- Updates the pre-processing step to omit null salaries to consider both base and extra pay, instead of the summed total pay
- Does a little Makefile reorganization to accommodate multiple years
- Removes legacy data and data-making

Closes #350.

## Notes

The [DataMade guide to Make](https://github.com/datamade/data-making-guidelines) and [this article demonstrating `csvkit`](https://source.opennews.org/articles/eleven-awesome-things-you-can-do-csvkit/) will be helpful when reviewing these changes! 

## Testing instructions

- Pull down changes: `git pull`.
- Check out this branch: `git checkout feature/hec/update-etl`
- Make the data: `docker-compose run --rm app make`. This may take a few minutes! Confirm the command finishes. Once it's finished, confirm you have four CSV files in the root of your project directory (three for 2017, one for 2018), and that each file contains data.
